### PR TITLE
cql authentication added

### PR DESCRIPTION
**Pull request by larisau**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -83,6 +83,9 @@ var (
 
 	measureLatency bool
 	validateData   bool
+
+	cqlUsername string
```
Comment (dahankzter) [inline main.go]: Maybe group them with the other `cql` properties like keyspaceName etc?

**Comment by dahankzter** [comment]
Small patch and we should merge it right away unless it destroys  future refactoring plans @rjeczalik ?

**Comment by denesb** [comment]
Should be documented in the `README.md`

File: main.go [hunk 2]
```diff
@@ -257,6 +260,10 @@ func main() {
 	flag.StringVar(&tableName, "table", "test", "table to use")
 
 	flag.StringVar(&hostSelectionPolicy, "host-selection-policy", "token-aware", "set the driver host selection policy (round-robin,token-aware,dc-aware),default 'token-aware'")
+
+	flag.StringVar(&cqlUsername, "cql-username", "", "cql username for authentication")
```
Comment (mmatczuk) [inline main.go]: Why would you want it to be prefixed with `cql`, how about just username and password - applies to both variables and flags.

Comment (larisau) [reply main.go]: ok, I can leave the cql word in the variable description only

**Comment by larisau** [comment]
@mmatczuk @dahankzter updated

**Pull request merged**